### PR TITLE
2021 Debts

### LINKED ISSUE #5202: tv100
**Channel Name:** tv100
**Country:** tr
**Language:** tr
**Website:** https://www.tv100.com/
**Stream URL (optional):** xxx
**Notes (optional):** livestream: https://www.tv100.com/canli-yayin

### LINKED ISSUE #5696: Add: Canal de la Ciudad
### Channel Name

Canal de la Ciudad

### Country

Argentina

### Language

Spanish

### Website

https://www.buenosaires.gob.ar/canaldelaciudad

### Stream URL

_No response_

### Notes

Embed frame: https://vmf.edge-apps.net/embed/live.php?streamname=gcba_video4-100042&autoplay=true2&autoplay=true

### LINKED ISSUE #5783: Add: Canal Luz
### Channel Name

Canal Luz

### Country

Argentina

### Language

Spanish

### Website

https://www.canalluz.org/index.html

### Stream URL

https://g2.vxral-slo.transport.edge-access.net/a11/ngrp:canal_luz01-100009_all/canal_luz01-100009_1080p.m3u8

### Notes

Christian/Religious TV station.

### LINKED ISSUE #5814: Add: Omega Tv
### Channel Name

Omega Tv

### Country

Honduras

### Language

Spanish

### Website

https://omegaturadio.com/

### Stream URL

https://5caf24a595d94.streamlock.net:1937/8142/8142/chunklist_w1325214426.m3u8

### Notes

Not 24/7

### LINKED ISSUE #5846: Add: Canal 5 (MX)
### Channel Name

Canal 5

### Country

México

### Language

Spanish

### Website

https://www.televisa.com/canal5/

### Stream URL

_No response_

### Notes

_No response_

### LINKED ISSUE #5847: Add: Azteca 7 (MX)
### Channel Name

Azteca 7

### Country

México

### Language

Spanish

### Website

https://www.tvazteca.com/azteca7/

### Stream URL

_No response_

### Notes

_No response_

### LINKED ISSUE #5886: Add: Channel 5
### Channel Name

Channel 5 (1080p)

### Country

Thailand

### Language

Thai

### Website

https://tv5.co.th/tv5hd1-web/page-live_streaming_tv5hd1.php

### Stream URL

http://110.170.117.27:1935/apptv5hd1live/vdo-tv5hd1/playlist.m3u8

### Notes

_No response_

### LINKED ISSUE #5941: Replace: Arte Deutsch
### I have verified that

- [X] I use the GitHub playlist link and not a downloaded/cached version of the file
- [X] My TV player has updated this playlist after 0:00 GMT and is set to update it daily
- [X] I am aware that the channel source may have temporal outages. It is broken/absent from the playlist for couple of hours or days

### Channel Name

Arte Deutsch

### What happened to the stream?

It is now offline (disappeared from the playlist)

### Playlist

https://iptv-org.github.io/iptv/channels.json

### Broken Link

https://artelive-lh.akamaihd.net/i/artelive_de@393591/master.m3u8?set-segment-duration=quality

### Possible Replacement

https://artesimulcast.akamaized.net/hls/live/2030993/artelive_de/master_v720.m3u8

### Notes

I live in Germany, so it's not a geo blocking issue.

### LINKED ISSUE #5961: TELE-MAG QUEBEC
### Channel Name

TELE-MAG QUEBEC

### Country

CANADA

### Language

FRENCH

### Website

https://tele-mag.tv/

### Stream URL

_No response_

### Notes

_No response_

### PR DESCRIPTION
**Pull request by Dum4G**
Closes iptv-org/iptv#5886
Closes iptv-org/iptv#5814
Closes iptv-org/iptv#5783
Closes iptv-org/iptv#5696
Closes iptv-org/iptv#5941
Closes iptv-org/iptv#5961
Closes iptv-org/iptv#5846
Closes iptv-org/iptv#5847
Closes iptv-org/iptv#5202

### CODE REVIEW

File: channels/mx.m3u [hunk 1]
```diff
@@ -195,3 +195,5 @@ https://5ca3e84a76d30.streamlock.net/gpacifico3/obregon.smil/playlist.m3u8
 http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/video/x3m1xfy
 #EXTINF:-1 tvg-id="UnoTV.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="" group-title="",Uno TV (720p) [Not 24/7]
 https://ooyalahd2-f.akamaihd.net/i/UnoTV01_delivery@122640/master.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="MX" tvg-language="Spanish" tvg-logo="" group-title="",Canal 5
+ http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.twitch.tv/5mxt291
```
Comment (LaneSh4d0w) [inline channels/mx.m3u:199]:
```suggestion
http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.twitch.tv/5mxt291
```

I suggest not adding it, as they change their name almost every week.

Comment (Dum4G) [reply channels/mx.m3u:199]: Yeah, name has already been changed

File: channels/tr.m3u [hunk 2]
```diff
@@ -463,3 +463,13 @@ https://waw2.artiyerelmedya.net/vuslattv/bant1/playlist.m3u8
...
 #EXTINF:-1 tvg-id="YolTV.tr" tvg-country="TR" tvg-language="Turkish" tvg-logo="" group-title="",Yol TV (720p) [Not 24/7]
 http://163.172.39.215:25461/live/line/C4@!a3a1@!w72A/107.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="TR" tvg-language="Turkish" tvg-logo="" group-title="",tv100 Ekonomi
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/watch?v=32enbX7XKnw
+#EXTINF:-1 tvg-id="" tvg-country="TR" tvg-language="Turkish" tvg-logo="" group-title="",tv100
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/watch?v=8jEXHzMTR7s
+#EXTINF:-1 tvg-id="" tvg-country="TR" tvg-language="Turkish" tvg-logo="" group-title="",tv100
+https://livex458745.livestreamlive.xyz/tv100.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="TR" tvg-language="Turkish" tvg-logo="" group-title="",Habitat TV Istanbul [Geo-blocked]
+http://istanbul.tvizlehd.com/live/habitattv.m3u8
```
Comment (LaneSh4d0w) [inline channels/tr.m3u:473]: 404 Not Found while testing on US IP

Comment (LaneSh4d0w) [reply channels/tr.m3u:473]: Same with Turkish IP

File: channels/tr.m3u [hunk 3]
```diff
@@ -463,3 +463,13 @@ https://waw2.artiyerelmedya.net/vuslattv/bant1/playlist.m3u8
...
+#EXTINF:-1 tvg-id="" tvg-country="TR" tvg-language="Turkish" tvg-logo="" group-title="",Habitat TV Ankara [Geo-blocked]
+https://ankara.tvizlehd.com/live/habitattv.m3u8
```
[the remark this answers is not in the document]
Comment (LaneSh4d0w) [reply channels/tr.m3u:475]: Alt (also dead) : https://izmir.tvizlehd.com/habitattv.m3u8

They all seem to be dead right now.

**Review by BellezaEmporium — DISMISSED** [review]

File: channels/hn.m3u [hunk 4]
```diff
@@ -79,3 +79,5 @@ https://emisoras.hn:8081/tvcopan/index.m3u8
 https://live-utv.unah.edu.hn/web/salida.m3u8
 #EXTINF:-1 tvg-id="WaldivisionInternacional.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="https://i.imgur.com/o5TnL2b.png" group-title="",Waldivisión Internacional (720p) [Not 24/7]
 https://tvdatta.com:3934/live/waldivisionlive.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="HN" tvg-language="Spanish" tvg-logo="" group-title="",Omega Tv
+https://5caf24a595d94.streamlock.net:1937/8142/8142/playlist.m3u8
```
Comment (freearhey) [inline channels/hn.m3u:83]: image

**Review by freearhey — CHANGES_REQUESTED** [review]

File: channels/hn.m3u (hunk 4 above)
Comment (Dum4G) [reply channels/hn.m3u:83]: image
Not 24/7

File: channels/hn.m3u [hunk 5]
```diff
@@ -79,3 +79,5 @@ https://emisoras.hn:8081/tvcopan/index.m3u8
 https://live-utv.unah.edu.hn/web/salida.m3u8
 #EXTINF:-1 tvg-id="WaldivisionInternacional.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="https://i.imgur.com/o5TnL2b.png" group-title="",Waldivisión Internacional (720p) [Not 24/7]
 https://tvdatta.com:3934/live/waldivisionlive.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="HN" tvg-language="Spanish" tvg-logo="" group-title="",Omega Tv
```
Comment (Dum4G) [inline channels/hn.m3u:82]:
```suggestion
#EXTINF:-1 tvg-id="" tvg-country="HN" tvg-language="Spanish" tvg-logo="" group-title="",Omega Tv [Not 24/7]
```

**Review by BellezaEmporium — APPROVED** [review]

**Review by freearhey — APPROVED** [review]

**Pull request merged**